### DISCUSSION TITLE
Https Issue

### DISCUSSION THREAD

**Discussion by CJRumbold** [NginxProxyManager/nginx-proxy-manager#2622] [General]
Hey there guys I'm trying to setup a ssl certificate on a server project I'm working on, sadly I've cocked up a bit and I've used the external IP to the server instead of the IP to Portainer. 

When I try to connect to my other Docker containers I get the "This connection is not secure" meaning I'm kinda soft locked out of my other containers. 

Anyone have any suggestions ?

**Comment by jeremy-chua** [comment]
This happens if public CA signed certificate is not used.
